content/10.diagnostics.md

```diff
@@ -1,33 +1,54 @@
 ## Diagnostics
 
+Identifying individuals who have contracted the infection is crucial to slow down the global pandemic.
 Given the high transmissibility of SARS-CoV-2, the development of reliable assays to detect SARS-CoV-2 infection even in asymptomatic carriers is vitally important.
 For instance, the deployment of wide-scale diagnostic testing followed by the isolation of infected people has been a key factor in South Korea's successful strategy for controlling the spread of the virus.
 Following the first release of the genetic sequence of the virus by Chinese officials on January 10 2020, the first test was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
-Identifying individuals who have contracted the infection is crucial to slowing down the global pandemic.
 The assays available to date use the following approaches to identify the active virus in patient samples.
 A range of diagnostic approaches from a methodological standpoint are being or could possibly be developed.
 
 ### Molecular Tests
 
-Molecular tests are often carried out from nasopharyngeal cavity samples, including throat wash and saliva [@doi:10/ggp4qx].
+Molecular tests are used to find specific sequences in a sample, which first requires getting a patient sample. 
+Samples used for molecular tests include nasopharyngeal cavity samples, including throat wash and saliva [@doi:10/ggp4qx], or stool samples [@doi:10.1002/jmv.25742].
+Molecular tests have false negative rates that depend on patient viral load, the testing target, and the sample type (nasopharyngeal swab, saliva, fecal, or otherwise.)
+Molecular testing has also been applied to broader community surveillance by testing wastewater. [@doi:10.1016/j.scitotenv.2020.138764]
+China has approved 11 different nucleic acid and reagent based tests [@url:http://www.nmpa.gov.cn/WS04/CL2056/375802.html].
+
+#### Amplification and Identification Processes
+Given a sample from a patient, tests require sample pre-processing, followed by library preparation, and then sequencing itself [doi:10.1016/j.biotechadv.2020.107537]. 
+Pre-processing is needed to copy the sample in order to create DNA from the RNA sample, then to produce quantities sufficient for a test by amplifying the DNA.
+Library preparation is the process of adding reagents that cut the sequences into fragments appropriate for use in a PCR machine [doi:10.1016/j.biotechadv.2020.107537].
 
 #### RT-PCR
 
+RT-PCR tests are a type of test which uses reverse transcriptase to translate the viral RNA into DNA for pre-processing and uses PCR testing for sequencing.
+There are different reagents used for library preparation which allow identifying one or more target section(s) with PCR.  [@doi:10.1021/acsnano.0c02624]
```

Comment:
```suggestion
PCR conditions must be optimized through the calibration of the protocol, the selection of reagents, etc. [@doi:10.1021/acsnano.0c02624].
```
This actually is not part of library prep, library prep is what you do before sequencing -- I'm rephrasing a bit here to make this distinction more clear